src/manager.js

```diff
@@ -417,16 +443,16 @@ AdManager.prototype.loadAds = function(element, updateCorrelator) {
  * @returns undefined
 */
 AdManager.prototype.refreshSlot = function(domElement) {
-  if ((domElement.getAttribute('data-ad-load-state') === 'loaded') || (domElement.getAttribute('data-ad-load-state') === 'loading')) {
-    return;
+  var that = this;
+  if (this.options.amazon_enabled && this.amznads) {
+      amznads.getAdsCallback('3706', function () {
```

Comment:
Another reason to suggest this should be a setting as this account number is `3706`, whereas it's `3076` correctly above.